cmd/traffic_crashlog/Makefile.am

```diff
@@ -20,6 +20,7 @@ bin_PROGRAMS = traffic_crashlog
 AM_CPPFLAGS = \
   $(iocore_include_dirs) \
   -I$(top_srcdir)/lib \
+  -I$(top_srcdir)/lib/ts \
```

Comment:
Why was this added back in? There was a bug specifically to get rid of that (TS-3759).